test/langtools/tools/javac/patterns/PrimitiveTypesInTestingContextErasure.java

```diff
@@ -0,0 +1,104 @@
+import java.util.List;
```

Comment:
nit: this import seems to be out of place

Reply:
Also moved it. Thx!